packages/noir-ethereum-api/src/noir/oracles/accountOracles.ts

```diff
@@ -14,6 +16,19 @@ export interface AccountWithProof {
   depth: string,
 }
 
+export interface EthBlock {
+  stateRoot: string
+}
```

Comment:
Duplicate with [BlockHeader](https://github.com/zkworks-xyz/noir-ethereum-history-api/blob/main/packages/noir-ethereum-api/src/ethereum/blockHeader.ts)

Reply:
Also you probably don't need it